CONTRIBUTING.md

```diff
@@ -222,6 +222,8 @@ If you are on Windows and not using WSL, you will have to use `rwt cp` (this is
 
 Also, you most likely first have to [install `rsync`](https://tlundberg.com/blog/2020-06-15/installing-rsync-on-windows/).
 
+Finally, ensure that appropriate Windows permissions are available to create symbolic links.  This can be accomplished via turning on Windows Developer Mode on Windows 10.  Go to Settings and find the `For developers` page.  Turn on the `Developer Mode` toggle.  Alternately, running your shell or VSCode as an Administrator will also work, but is the less preferred option.
```

Comment:
```suggestion
Finally, ensure that appropriate Windows permissions are available to create symbolic links. This can be accomplished via turning on Windows Developer Mode on Windows 10. Go to Settings and find the `For developers` page. Turn on the `Developer Mode` toggle. Alternately, running your shell or VSCode as an Administrator will also work, but is the less preferred option.
```

I thought "double space after period" was what you were supposed to do when writing English. So I had to ask when I started contributing. But I was told we don't use that writing style in the RW docs.